guides/code-style/.rubocop.yml

```diff
@@ -1,10 +1,16 @@
-Metrics/LineLength:
-  Max: 80
+AllCops:
+  DisplayCopNames: true
   Exclude:
     - 'db/schema.rb'
+    - 'bin/*'
+    - 'vendor/*'
+    - './Rakefile'
+
+Metrics/LineLength:
+  Max: 80
 
 Metrics/MethodLength:
-  Enabled: false
+  Max: 15
```

Comment:
`Max: 10` is enough

Reply:
I think 10 is to small. How about 12 lines?